scripts/qa_crowbarsetup.sh

```diff
@@ -4321,12 +4321,41 @@ function onadmin_testpreupgrade
     oncontroller testpreupgrade
 }
 
+function oncontroller_get_fips
+{
+    # checking for cloudver might not get correct results here due to the nature of upgrade
+    local version=`openstack --version 2>&1 | cut -f 2 -d ' '`
+    if [[ "$version" < "3" ]]; then
```

Comment:
Hm. That actually does not hurt in this case, as we have 1.7.2 version on cloud6 side and 3.something on cloud7.

`"$version" -lt "3" ` actually fails with that `1.7.2`